com.seventhstring.Transcribe.yml

```
@@ -0,0 +1,61 @@
+app-id: com.seventhstring.Transcribe
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/transcribe/transcribe
+rename-desktop-file: seventhstring-transcribe.desktop
+rename-icon: seventhstring-transcribe
+finish-args:
+#TODO: figure out if its possible to only grant permissions 
+# on ~/.Transcribe!7 and xdg-music:ro, for now we use home
+# as a functional compromise
+- --share=ipc
+- --socket=x11
+- --socket=fallback-x11
```

This is not what I said.

---

```suggestion
- --socket=fallback-x11
- --socket=wayland
```

The binary links against Gtk3 so this should work.

---

Oh sorry, i misunderstood! What did you mean?

I just had a look at a few other flatpaks i have installed and 
```
  - --share=ipc
  - --socket=x11
  - --socket=wayland
```
seems to be a fairly common configuration. should i go with that?

---

works just fine, can i drop `--share=ipc` too then?

---

I never mentionned it.

NO

---

> I never mentionned it.
> 
> NO

ok! i have read somewhere that its needed for `socket=x11` permissions, so thats why i asked!

---

the application does not seem to support wayland. running with `wayland + fallback-x11` on my local system ( using wayland ) gives: `13:34:59: Error: Unable to initialize GTK+, is DISPLAY set properly?`. running just with `x11` works fine, so thats what im inclined to do after some testing.